cdist/conf/type/__jail_freebsd10/gencode-remote

```diff
@@ -111,6 +111,7 @@ startJail() {
    fi
 }
 
+# shellcheck disable=SC2120
```

Comment:
This error is legitimate (and is caused by a bug).
In line 125 and line 130 the code mentions `$3` which AFAICS is expanded by the heredoc, not by awk.